testutils/goroutines/stacks.go

```diff
@@ -53,15 +54,14 @@ func (s Stack) Full() []byte {
 }
 
 func (s Stack) String() string {
-	return fmt.Sprintf("Goroutine %v in state %v:\n%s", s.id, s.state, s.Full())
+	return fmt.Sprintf("Goroutine %v in state %v first func: %v:\n%s", s.id, s.state, s.firstFunction, s.Full())
```

Comment:
nit: could we make this message slightly better, without the double colons?

Perhaps `"Goroutine %v in state %v, with %v on top of the stack:\n%s"`?